### PR TITLE
Portuguese translation - debugging

### PR DESCRIPTION
**Pull request by andreregino**

### CODE REVIEW

File: pt/development/debugging.rst [hunk 1]
```diff
@@ -1,13 +1,183 @@
...
+desenvolvimento. Ainda que o CakePHP não forneça nenhuma ferramenta que se conecte 
+com qualquer IDE ou editor de texto, este oferece várias ferramentas que
+auxiliam na depuração e exibição de tudo que está sendo executado "por baixo dos panos"
+na sua aplicação.
 
-    Por favor, sinta-se a vontade para nos enviar um pull request no
-    `Github <https://github.com/cakephp/docs>`_ ou use o botão
-    **Improve This Doc** para propor suas mudanças diretamente.
+Depuração Básica
+===============
```
Comment (markstory) [inline pt/development/debugging.rst]: Underlines must be the same length as the text.

File: pt/development/debugging.rst [hunk 2]
```diff
@@ -1,13 +1,183 @@
...
+    // Saídas
+    object(Car) {
+        color => 'red'
+        make => 'Toyota'
+        model => 'Camry'
+        mileage => (int)15000
+    }
+
+Criando Logs com Pilha de Execução
+=========================
```
Comment (markstory) [inline pt/development/debugging.rst]: Underlines must be the same length as the text.

File: pt/development/debugging.rst [hunk 3]
```diff
@@ -1,13 +1,183 @@
...
+.. php:staticmethod:: log($var, $level = 7, $depth = 3)
+
+Cria um log detalhado da pilha de execução no momento em que a mesma foi invocada. O
+método ``log()`` exibe dados similares ao``Debugger::dump()``, 
+mas no arquivo debug.log ao invés do buffer de saída principal. 
+É valido ressaltar que o diretório **tmp** e seu conteúdo devem ter 
+permissão de escrita para o servidor web a fim de que a função ``log()`` consiga executar corretamente.
+
+Gerando Pilhas de Execução
+=======================
```
Comment (markstory) [inline pt/development/debugging.rst]: Underlines must be the same length as the text.

File: pt/development/debugging.rst [hunk 4]
```diff
@@ -1,13 +1,183 @@
...
+Ainda que este método seja usado internamente, o mesmo pode ser conveniente 
+caso você esteja criando suas próprias mensagens de erros e registros de logs.
+
+.. php:staticmethod:: Debugger::getType($var)
+
+Obtém o tipo da variável. Caso seja um objeto, o retorno do método será o nome de sua classe
+
+
+Usando Logging para Depuração
+======================
```
Comment (markstory) [inline pt/development/debugging.rst]: Underlines must be the same length as the text.

**Comment by cake17** [comment]
@andreregino thanks for your work ! I'll correct the sphinx issues post merge.

**Pull request merged**